test/unit/components/app/app_spec.js

```diff
@@ -16,15 +17,15 @@ describe('Unit: Component: App', function() {
 
   beforeEach(function() {
     
-    angular.mock.inject(($compile, $rootScope) => {
+    angular.mock.inject(($compile, $rootScope, $httpBackend) => {
```

Comment:
`$httpBackend` is not being used now. Do you plan to rewrite this test case in the near future?